generators/generator-base.js

```diff
@@ -167,6 +167,16 @@ module.exports = class JHipsterBaseGenerator extends PrivateBase {
     return this.fetchFromInstalledJHipster(this._jhipsterGenerator, 'templates', ...args);
   }
 
+  /**
```

Comment:
The file above `generators/generate-blueprint/templates/generators/generator/templates/template-file.ejs` is empty, is it correct?

Reply:
Permission change 755 -> 644